react/features/filmstrip/functions.web.js

```diff
@@ -1,10 +1,20 @@
 // @flow
 
+import { JitsiParticipantConnectionStatus } from '../base/lib-jitsi-meet';
+import { MEDIA_TYPE } from '../base/media';
 import {
+    getLocalParticipant,
+    getParticipantById,
     getParticipantCountWithFake,
     getPinnedParticipant
 } from '../base/participants';
 import { toState } from '../base/redux';
+import {
+    getLocalVideoTrack,
+    getTrackByMediaTypeAndParticipant,
+    isLocalTrackMuted,
+    isRemoteTrackMuted
+} from '../base/tracks';
```

Comment:
Could you please give me more info about this? 

Why do you want me to do this change? Those are actually defined in /functions and I see we have index.js that is exporting them. I'm a little bit confused.

What other imports? All new imports in the PR or all the imports in the file or all 'base/tracks' imports in the project?

Reply:
In order to avoid circular imports (see the webpack warnings for example), which often break mobile, I started importing actions from /actions, functions from /functions and so on. The end goal is to eventually drop those index.js files altogether.

An innocent import for a single function triggers importing components, atlaskit, and so on, which is very often the source of web breaking mobile.

Reply:
Fixed!